tests/test_diff.py

```diff
@@ -88,7 +88,7 @@ def test_diff_points(
                 "+                                     geom = POINT(...)",
                 "+                                  t50_fid = 9999999",
                 "+                               name_ascii = Te Motu-a-kore",
-                "+                               macronated = 0",
+                "+                               macronated = N",
```

Comment:
did this change types?

Reply:
It did not - it has always been a VARCHAR(1), and in the test data, it is filled with only "N" and "Y".
The test-helper tries to insert a `False` into this column, which in GPKG land, works okay - it gets converted to "0". But in Postgis land it wasn't happy.
I could have split the test-helper into two variants so we put False into it in GPKG, and "N" in Postgis, but it seemed better just to fix the test-helper so it doesn't rely on auto type conversion.